src/components/Button/index.stories.js

```diff
@@ -1,21 +1,18 @@
 import React from 'react';
 import { storiesOf } from "@storybook/react";
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
-import { faMapMarker } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faMapMarker } from '@fortawesome/free-solid-svg-icons';
+import { faChevronRight } from '@fortawesome/free-solid-svg-icons';
```

Comment:
```suggestion
import { faMapMarker, faChevronRight } from '@fortawesome/free-solid-svg-icons';
```
Tu peux sortir les deux icônes depuis le même import